### ISSUE TITLE
[Security Leak] Check for log4j version

### ISSUE THREAD

**Issue by JuKu** [JuKu/ansible-manager#13]
Currently many newspaper write about the big log4j security leak, which allows Remote Code Execution.
We have to check, if out current used log4j version is also affected.

**Additional information**
- https://www.heise.de/ratgeber/Schutz-vor-schwerwiegender-Log4j-Luecke-was-jetzt-hilft-und-was-nicht-6292961.html
- https://www.heise.de/news/Roter-Alarm-Log4j-Zero-Day-Luecke-bedroht-Heimanwender-und-Firmen-6292863.html
- https://www.heise.de/news/Log4j-2-16-0-verbessert-Schutz-vor-Log4Shell-Luecke-6294053.html
- https://www.heise.de/meinung/Kommentar-zu-log4j-Es-funktioniert-wie-spezifiziert-6294476.html

**Comment by JuKu** [comment]
State: The version has the following log4j dependencies:
```java
org.apache.logging.log4j:log4j-to-slf4j:jar:2.14.1:compile
[INFO] |  |  |  |  \- org.apache.logging.log4j:log4j-api:jar:2.14.1:compile
```

This means the current ansible manager version is **affected**, because the spring contains this dependencies.
This means, that we have to upgrade spring version, see also #15 .

**Comment by JuKu** [comment]
Was fixed by #18 and the newest commít.
